cpiface/zmq-cpiface.cc

```diff
@@ -242,7 +242,7 @@ int main(int argc, char **argv) {
             BessClient b(CreateChannel(std::string(args.bessd_ip) + ":" +
                                            std::to_string(args.bessd_port),
                                        InsecureChannelCredentials()));
-            b.runRemoveCommand(rbuf.sess_entry.ue_addr.u.ipv4_addr,
+            b.runRemoveCommand(ntohl(rbuf.sess_entry.ue_addr.u.ipv4_addr),
```

Comment:
Yes we saw this issue the other day looking at the logs where delete session was reversed . Why is delete a special case?

Reply:
Is there consistency issue to be fixed in the CP?

Reply:
Most likely. Yes. Not sure why this wasn't patched up in the CP.

Reply:
lets open an issue in CP if there is an inconsistency there and link it in a comment so we don't forget.